\hypertarget{classExceptions_1_1UsersError}{}\doxysection{Exceptions.\+Users\+Error Class Reference}
\label{classExceptions_1_1UsersError}\index{Exceptions.UsersError@{Exceptions.UsersError}}


Inheritance diagram for Exceptions.\+Users\+Error\+:
% FIG 0


Collaboration diagram for Exceptions.\+Users\+Error\+:
% FIG 1
\doxysubsection*{Public Member Functions}
\begin{DoxyCompactItemize}
\item 
None \mbox{\hyperlink{classExceptions_1_1UsersError_ad4a39d5a3707557ac7f499eea1acc116}{\+\_\+\+\_\+init\+\_\+\+\_\+}} (self, str error\+\_\+message, $\ast$$\ast$kwargs)
\end{DoxyCompactItemize}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Exception to raise when a user's opi call produces an error.
\end{DoxyVerb}
 

\doxysubsection{Constructor \& Destructor Documentation}
\mbox{\Hypertarget{classExceptions_1_1UsersError_ad4a39d5a3707557ac7f499eea1acc116}\label{classExceptions_1_1UsersError_ad4a39d5a3707557ac7f499eea1acc116}} 
\index{Exceptions.UsersError@{Exceptions.UsersError}!\_\_init\_\_@{\_\_init\_\_}}
\index{\_\_init\_\_@{\_\_init\_\_}!Exceptions.UsersError@{Exceptions.UsersError}}
\doxysubsubsection{\texorpdfstring{\_\_init\_\_()}{\_\_init\_\_()}}
{\footnotesize\ttfamily  None Exceptions.\+Users\+Error.\+\_\+\+\_\+init\+\_\+\+\_\+ (\begin{DoxyParamCaption}\item[{}]{self,  }\item[{str}]{error\+\_\+message,  }\item[{$\ast$$\ast$}]{kwargs }\end{DoxyParamCaption})}

\begin{DoxyVerb}Initialize a user's error.
:param error_message: Message explaining the error.
:param kwargs: Any additional key word arguments.
\end{DoxyVerb}
 

Reimplemented from \mbox{\hyperlink{classExceptions_1_1PapertrailError_ae940a760b85731141ff662dd8e104750}{Exceptions.\+Papertrail\+Error}}.



The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
Exceptions.\+py\end{DoxyCompactItemize}
